### PR TITLE
Diagnostic Cards

### PR DESCRIPTION
**Pull request by dinadeljanin**
Screen Shot 2020-03-30 at 04 05 30

The card structure is complete. The only thing missing from it, rather, is that the checkbox is still there. I'm trying to figure out how to tie the label's current background to the checkbox when it's been checked.

### CODE REVIEW

File: client/src/css/App.css [hunk 1]
```diff
@@ -150,4 +150,4 @@
 	100%{
 		-moz-transform:rotate(360deg)
 	}
-}
\ No newline at end of file
+}
```
Comment (AdhamAH) [inline client/src/css/App.css]: it looks great. Is this by accident?

**Comment by AdhamAH** [comment]
I love it by the way!

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```
Comment (SomeMoosery) [inline yarn.lock]: Please remove from root directory

File: client/src/css/App.css (hunk 1 above)
Comment (dinadeljanin) [reply client/src/css/App.css]: Awesome! And yes yes. This is where I was testing out the CSS.

**Comment by lukelin1991** [comment]
whats "rebase off master" mean? o.0

**Comment by SomeMoosery** [comment]
Closing because #311 created a separate PR for some reason :|

**Pull request closed without merging**